subprojects/build-init/src/main/java/org/gradle/buildinit/plugins/internal/modifiers/Language.java

```diff
@@ -30,15 +32,15 @@
     private final String extension;
 
     public static Language withName(String displayName) {
-        return new Language(displayName.toLowerCase(), displayName, displayName.toLowerCase());
+        return new Language(TextUtil.toLowerCaseUserLocale(displayName), displayName, TextUtil.toLowerCaseUserLocale(displayName));
     }
 
     public static Language withName(String displayName, String name) {
         return new Language(name, displayName, name);
     }
 
     public static Language withNameAndExtension(String displayName, String extension) {
-        return new Language(displayName.toLowerCase(), displayName, extension);
+        return new Language(TextUtil.toLowerCaseUserLocale(displayName), displayName, extension);
     }
```

Comment:
I think these need to be toLowerCaseLocaleSafe too.  The language names are expected to be exactly as we have them above